frontend/Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM registry.redhat.io/rhel8/python-38
-LABEL authors="Aberson Malivert(amaliver@redhat.com),Ashley Kim(akim@redhat.com, Julia Walker(juwalker@redhat.com), Pablo Castillo(pcastill@redhat.com), Sarah Zhong(szhong@redhat.com)
+LABEL authors="Aberson Malivert(amaliver@redhat.com),Ashley Kim(akim@redhat.com, Julia Walker(juwalker@redhat.com), Pablo Castillo(pcastill@redhat.com), Sarah Zhong(szhong@redhat.com)"
 USER root 
 RUN dnf install -y python3-pip
 RUN dnf -y install gpgme-devel
 RUN python3 -m pip uninstall insights-client
 RUN cd /usr/local/bin/
-RUN git clone https://github.com/amalivert/certification_notifier.git && cd certification_notifier && git checkout dev && python3 -m pip uninstall -r requirements.txt && python3 -m pip install -r requirements.txt %% chmod u+x rh_cert_notify.py
-CMD [ "python3", "./rh_cert_notify.py"]
+RUN git clone https://github.com/amalivert/certification_notifier.git && cd certification_notifier && git checkout dev && cd frontend && python3 -m pip uninstall -r requirements.txt && python3 -m pip install -r requirements.txt %% chmod u+x rh_cert_notify.py
```

Comment:
If you combine these 8 lines into 3 lines, you'll have less layers and decrease the docker image size.